modules/seqexec/engine/src/test/scala/seqexec/engine/packageSpec.scala

```diff
@@ -204,30 +204,6 @@ class packageSpec extends FlatSpec with NonImplicitAssertions {
     assert(result.isDefined)
   }
 
-  "engine" should "not capture runtime exceptions." in {
```

Comment:
This test is important, to ensure we are not discarding relevant errors.

Reply:
There is a reason for this. The engine needs to do `F[A].attempt` to capture regular errors. Almost by definition that will capture runtime exceptions. If I recall this test was about not capturing things like OOM, perhaps we should change it that way?

Reply:
I mean change the test to not capture e.g OOM